app/serializers/error_serializer.rb

```diff
@@ -3,8 +3,8 @@
 class ErrorSerializer
   include FastJsonapi::ObjectSerializer
 
-  set_id do |err, _params|
-    "#{err.class.name}_#{SecureRandom.hex(20)}"
+  set_id do |_err, _params|
+    DateTime.now.to_i.to_s
```

Comment:
Why do we changing id? With DateTime.now there is this super-small chance that we get the same id (even though, we don't store it anywhere :))

Reply:
You are right here! 
I will back to previous version 👍